cpg_infra/config/config.py

```diff
@@ -123,6 +123,10 @@ class GCP(DeserializableDataclass):
         gcp: GCP
         slack_channel: str | None = None
         etl_accessors: list[str] = dataclasses.field(default_factory=list)
+        etl_environment: str | None = None
+        etl_default_config: dict[str, str] | None = None
```

Comment:
Can you add a couple of comments describing what values you'd expect in these fields?

Reply:
Comments and example values added, I had renamed config to 'etl_parser_default_config' to make the name clearer.